miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -301,6 +301,45 @@ var Utils = Utils || {
             }
         });
     },
+    printDialog: function(type, ids) {
+        Utils
+            .ajaxWithDialog(
+                'Getting Printers',
+                'GET',
+                window.location.origin + '/miso/rest/printer',
+                null,
+                function(printers) {
+                  Utils
+                      .showWizardDialog(
+                          'Select Printer',
+                          printers
+                              .filter(function(printer) {
+                                return printer.available;
+                              })
+                              .map(
+                                  function(printer) {
+                                    return {
+                                      name : printer.name,
+                                      handler : function() {
+                                        Utils
+                                            .ajaxWithDialog(
+                                                'Printing',
+                                                'POST',
+                                                window.location.origin + '/miso/rest/printer/' + printer.id + '?' + jQuery
+                                                    .param({
+                                                      type : type,
+                                                      ids : ids.join(',')
+                                                    })
+                                                , null, function(result) {
+//TODO: replace with Utils.showOkDialog
+alert(result == ids.length ? 'Printing successful.' : (result + ' of ' + ids.length + ' printed.'));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)